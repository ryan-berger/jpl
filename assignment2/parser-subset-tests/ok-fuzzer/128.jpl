print ""
time show R8V_9F_7b_7v.0M.8M.()
time read image "" to K0Z_
assert 477, "?YkK"
let z2h. = 9
read image ">a$;" to J
