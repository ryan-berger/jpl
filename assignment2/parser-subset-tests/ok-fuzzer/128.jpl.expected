(PrintCmd "")
(TimeCmd (ShowCmd (CallExpr R8V_9F_7b_7v.0M.8M.)))
(TimeCmd (ReadImageCmd "" (VarArgument K0Z_)))
(StmtCmd (AssertStmt (IntExpr 477) "?YkK"))
(StmtCmd (LetStmt (ArgLValue (VarArgument z2h.)) (IntExpr 9)))
(ReadImageCmd ">a$;" (VarArgument J))
